test/integration/features/cmd-config.feature

```diff
@@ -210,17 +210,8 @@ user defined options which changes default behaviour of Minishift.
      Then stdout should match "Disk size\s*:\s*25 GB"
      Then stdout should match "vCPUs\s*:\s*3"
       And Minishift should have state "Running"
-
-  Scenario: Checking that disk-size value was applied
-     When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
-     Then stdout should match "Disk \/dev\/.da: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
-
-  Scenario: Checking that cpus value was applied
-     When executing "minishift ssh -- cat /proc/cpuinfo" succeeds
-     Then stdout should match "processor\s*: 0"
-      And stdout should match "processor\s*: 1"
-      And stdout should match "processor\s*: 2"
-      And stdout should not match "processor\s*: [3-9]"
+      And the vm should run with 3 cpus
```

Comment:
maybe adding quotes would make the number more visible and signal that it is a parameter: `And the vm should run with "3" cpus`

Reply:
Done

Reply:
> it is possible there will be differences on different hypervisors, then this should take an interval, for example:
And the vm should run with "25" to "26" GB of disk size

Nothing is impossible ;)
I do not see any major benefit in putting the range in the step. Moreover this is not a minishift specific use case, so can be compromised